tests/spread/large/init/task.yaml

```diff
@@ -6,7 +6,7 @@ execute: |
   run_rockcraft pack
 
   test -f my-rock-name_0.1_amd64.rock
-  test ! -d work
+  test ! -d parts -a ! -d stage -a ! -d prime
```

Comment:
These tests could be a bit more readable if I moved this check into a new file and function called `tools/spread/is_work_dir_clean`